.changeset/smart-pillows-wave.md

```diff
@@ -0,0 +1,5 @@
+---
+"sit-onyx": minor
+---
+
+Implement loading state
```

Comment:
```suggestion
Add property `loading` for OnyxButton, OnyxCheckbox, OnyxRadioButton and OnyxSwitch
```

We should be as detailed as possible for the changelogs so users know what happened :)